src/components/Avatar.less

```diff
@@ -0,0 +1,9 @@
+.Avatar {
+  display: block;
+  border-radius: 8px;
```

Comment:
And here we can change `border-radius: 20%`, so it will adapt on any avatar size.